docs/db/changelog/changesets/user-data-store/0.1.x/20230322-audit-indexes.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.9.xsd">
+
+    <changeSet id="1" logicalFilePath="user-data-store/0.1.x/20230322-audit-indexes.xml" author="Zdenek Cerny">
+        <comment>Create indexes on table audit_log and audit_param</comment>
```

Comment:
Missing `exists precondition`. Moreover, each index should be extra changeset. See https://github.com/wultra/enrollment-server/blob/develop/docs/db/changelog/changesets/enrollment-server/1.4.x/20230314-audit.xml